lib/observer.js

```diff
@@ -20,12 +20,28 @@ function ObserverMixin() {
 
 /**
  * Register an asynchronous observer for the given operation (event).
+ *
+ * Example:
+ *
```

Comment:
If you can add a description before the code, it'll help users understand what it does.